_posts/2017-06-27-Increasing-Rusts-Reach.md

```diff
@@ -0,0 +1,310 @@
+---
+layout: post
+title: "Increasing Rust’s Reach"
+author: Carol Nichols
+---
+
+One of [Rust's 2017 goals](https://blog.rust-lang.org/2017/02/06/roadmap.html) is to make it easier
+to become productive in Rust by reducing its learning curve. We want Rust to be accessible to as
```

Comment:
> We want Rust to be accessible to as many people as possible because Rust is an enabling technology that can unlock systems programming for more people.

This phrasing is a bit awkward. How about:

We believe Rust has potential as an enabling technology, empowering a people who would not traditionally do system programming to take it on with confidence.